billboard.py

```diff
@@ -314,7 +324,43 @@ def getMinistatsCellValue(fieldName, ifNoValue=None):
             self.entries.append(entry)
 
     def _parseNewStylePage(self, soup):
-        dateElement = soup.select_one("button.date-selector__button.button--link")
+        artists = []
+        # extract artists info from page
+        pageContent = json.loads(soup.select_one("div#charts")['data-charts'])
+        for artist in pageContent:
+            artistName = artist['artist_name']
+            artistImages = artist['artist_images']
+            titleImages = artist['title_images']
+            artistImage = None
+            titleImage = None
+            # searching for image in 3 different sizes
+            if artistImages:
+                _artistImage = artistImages['sizes'].get('original')
+                if not _artistImage:
```

Comment:
I think

````
x = z1
if not x:
  x = z2
if not x:
  x = z3
````

is usually written `x = z1 or z2 or z3`.

Reply:
although it wouldn't be good for readability , don't you think ?
also, it is possible to add more later. Since sometimes the image is available only in small sizes

Reply:
What about this?

````python
_artistImage = None
for variant in ['original', 'ye-landing-lg-2x', 'ye-landing-med-2x']:
  _artistImage = _artistImage or artistImages['sizes'].get(variant)
````

Reply:
this will do it !